src/olympia/devhub/tests/test_feeds.py

```diff
@@ -37,7 +37,8 @@ def log_updates(self, num, version_string='1'):
         version = Version.objects.create(version=version_string, addon=self.addon)
 
         for i in range(num):
-            ActivityLog.create(amo.LOG.ADD_VERSION, self.addon, version)
+            with core.override_remote_addr('127.0.0.1'):
```

Comment:
Why is this override needed if there is no assertion about the value of the IP address?

Reply:
By default there is no value, but since this creates an `ActivityLog` with `ADD_VERSION` this would trigger the code I've added and fail. The overrides just ensures the call goes through.